icu4c/source/i18n/reldatefmt.cpp

```diff
@@ -817,6 +839,9 @@ RelativeDateTimeFormatter::RelativeDateTimeFormatter(
     } else {
         init(nfToAdopt, nullptr, status);
     }
+#else
```

Comment:
```
#else
     if (capitalizationContext == UDISPCTX_CAPITALIZATION_FOR_BEGINNING_OF_SENTENCE) {
        status =   U_UNSUPPORTED_ERROR;
        return ;
    }
    init(nfToAdopt, status);
#endif // !UCONFIG_NO_BREAK_ITERATION
```

Reply:
I agree. Modification: I would move the `#if !UCONFIG_NO_BREAK_ITERATION` inside the `if (capitalizationContext == ...) {`

Reply:
Fixed.